core/server/worker/src/test/java/alluxio/worker/file/FileDataManagerTest.java

```diff
@@ -103,7 +103,8 @@ public void persistFile() throws Exception {
     writeFileWithBlocks(fileId, blockIds);
 
     // verify file persisted
-    assertEquals(Arrays.asList(fileId), mManager.getPersistedFiles());
+    FileDataManager.PersistedFilesInfo info = mManager.getPersistedFilesInfo();
+    assertEquals(Arrays.asList(fileId), info.idList());
```

Comment:
can we verify the ufs statuses as well?

Reply:
The ufs is mocked, so it will have a `null` status. Is checking for `null` useful?